.github/CODEOWNERS

```diff
@@ -105,18 +105,18 @@
 /lib/DriverTool/swift_symbolgraph_extract_main.cpp  @QuietMisdreavus
 /lib/Frontend/*ModuleInterface*                     @artemcm @tshortli
 # TODO: /lib/IRGen/
-/lib/IDE/                                           @ahoppen @bnbarham @rintaro @hamishknight
-/lib/IDETool/                                       @ahoppen @bnbarham @rintaro @hamishknight
-/lib/Index/                                         @bnbarham
-/lib/Refactoring/                                   @ahoppen @bnbarham
-/lib/IRGen/*Coverage*                               @hamishknight
+/lib/IDE/                                           @ahoppen @bnbarham @hamishknight @rintaro
+/lib/IDETool/                                       @ahoppen @bnbarham @hamishknight @rintaro
+/lib/Index/                                         @ahoppen @bnbarham @hamishknight @rintaro
+/lib/Refactoring/                                   @ahoppen @bnbarham @hamishknight @rintaro
+/lib/IRGen/*Coverage*                               @ahoppen @bnbarham @hamishknight @rintaro
```

Comment:
May I take the opportunity to ask you to fix the order of these patterns? `IRGen` should go before `Index` and `Refactoring` should be moved further down.

Reply:
Sure, done 👍